tests/core/templates/core/admin/change_list.html

```diff
@@ -0,0 +1,6 @@
+{% extends "admin/change_list.html" %}
```

Comment:
My view is that having the custom element rendered as an inactive button will confuse users.  My suggestion is to change this so that it is a hidden element, and then add a comment explaining why it is required.

An example is:

```html
{% extends "admin/change_list.html" %}
{% comment %}
A custom template used for testing additions to the change_list page (See #1483).
{% endcomment %}
{% block result_list %}
  {{ block.super }}
  <p class="hidden">Custom change list item</p>
{% endblock %}

```

Reply:
Great suggestion, thanks! This is now updated, the actual rendering in the browser is not affected anymore:

![Screenshot 2022-09-05 at 14 28 23](https://user-images.githubusercontent.com/2872635/188460736-b85021d6-d512-42a4-86ec-f2b6ef812568.png)